### PR TITLE
refactor(api): update GOOGLE_OAUTH2 env variables

### PR DESCRIPTION
**Pull request by Samy-33**
Updated from GOOGLE_OAUTH2_KEY to GOOGLE_OAUTH2_CLIENT_ID and GOOGLE_OAUTH2_SECRET to GOOGLE_OAUTH2_CLIENT_SECRET  to make it consistent with Google keywords and the UI.

### CODE REVIEW

File: api/.env.template [hunk 1]
```diff
@@ -3,5 +3,5 @@ ENV=dev
 CORS_ALLOWED_ORIGINS=http://localhost:3000,http://127.0.0.1:3000
 ALLOWED_HOSTS=localhost,127.0.0.1
 DJANGO_SETTINGS_MODULE=api.settings.dev
-GOOGLE_OAUTH2_KEY=<replace with your own key>
+GOOGLE_OAUTH2_CLIENT_ID=<replace with your own key>
 GOOGLE_OAUTH2_SECRET=<replace with your own secret>
```
Comment (PranshuNayak) [inline api/.env.template:7]: This can also be changed to `GOOGLE_OAUTH2_CLIENT_SECRET`

Comment (Samy-33) [reply api/.env.template:7]: Good point.

**Review by PranshuNayak — APPROVED** [review]
LGTM

**Review by Aksh-Bansal-dev — APPROVED** [review]
LGTM.

**Pull request merged**